README.md

```diff
@@ -1,38 +1,56 @@
-# Cloud Native Networking Preamble
+# Cloud Native Principles
 
-**What is a Cloud Native Network Function \(CNF\)?**
+To talk about cloud native applications and systems, we need to define cloud native.  CNCF provides a high-level overview of what it means to be cloud native and lists some implementations and the benefits they provide in the [CNCF Cloud Native Definition v1.0 defintion](https://github.com/cncf/toc/blob/master/DEFINITION.md).
 
-In order to talk about CNFs, we need to define [cloud native](cloud-native-principles.md) \[1\]. Cloud native systems are, among other things, a set of loosely coupled services. These services, also known as microservices, are deployed onto immutable infrastructure while being managed by an orchestrator. This paper includes four links to other papers that go into detail about the definitions of cloud native, microservices, immutable infrastructure, and CNFs from an OSI layer perspective.
+The [cloud native principles](https://github.com/cloud-native-principles/cloud-native-principles) seek to clarify further the underlying fundamental concepts of what it means to be cloud native.
 
-**How are** [**cloud native**](cloud-native-principles.md) **systems loosely coupled?**
+NOTE: This is a work in progress.
 
-Cloud native systems have a clear separation between their processes \[2\]. They utilize the Unix philosophy of doing one thing and doing it well. These microservices usually use a technology like containers and aim for one process per container \[3\]. As such, cloud native applications should have all of their dependencies packaged with them during the build phase and leveraged during deployment \[4\].
+## Principles
 
-**How is** [**immutable infrastructure**](cloud-native-immutable-infrastructure-principles.md) **provisioned?**
+- Service discovery
+- Loosely coupled services
+- Microservices
+- Low overhead/coarse-grained packaging 
+   - examples: containers, unikernel
+- Declarative consumption model
+   - includes APIs and configuration
+   - applies to all layers: infra, workloads, platform, and components
+- Immutable infrastructure 
+- Designed for automation
 
-[Immutable infrastructure](cloud-native-immutable-infrastructure-principles.md) \(the orchestrator and all of the software and hardware that it depends on\) is provisioned using **baked** and **versioned** **templates** \[5\] \(e.g. server images produced from packer \[10\]\) or a combination of templates and **bootstrapping** \[6\] \(some repeatable and versioned process that is applied to the template e.g. Kubeadm\). The underlying infrastructure is not changed after it is made ready for use. New changes to the infrastructure are rolled out as new instances of infrastructure.
+Each of these concepts is covered in more depth in the [cloud-native-principles.md paper](https://github.com/cloud-native-principles/cloud-native-principles/blob/master/cloud-native-principles.md).
```

Comment:
Might want to separate take "paper" out of the link.

[cloud-native-principles.md](https://github.com/cloud-native-principles/cloud-native-principles/blob/master/cloud-native-principles.md) paper.

Reply:
updated